Webapp/sources/services/queries/compound.py

```diff
@@ -23,3 +23,14 @@ def get_compound_data_error_reports() -> List[models.CompoundDataErrorReport]:
     """
     # Add time cut off if list grows too large
     return db.session.query(models.CompoundDataErrorReport).all()
+
+
+def get_smiles_from_primary_key(primary_key: int) -> str:
```

Comment:
```suggestion
def get_smiles(primary_key: int) -> str:
```

Reply:
Cheers Andy, much appreciated for these suggested changes!